docs/pages/Theming/CustomTheme.md

```diff
@@ -2,139 +2,174 @@
 
 One of the powerful aspects of FURN theming is that it is heavily customizable. We recognize that users may have a need to change various parts of default themes, so we allow for several ways to specify overrides.
 
-You can create a custom theme by either building on top of an existing `ThemeReference` or building one from scratch. This then can be used by [passing it into our `ThemeProvider`](./Basics.md), which makes the theme available for all child components of the provider.
+## Creating a custom theme
 
-## Different aspects of customization
+There are two ways to make a custom theme: you can tack onto an existing one using `ThemeRecipes`, or create one from scratch. Generally it's easier to extend one of the default ones we provide.
 
-There's two main ways to customize a FURN theme - you can specify how to customize components, or you can customize theme tokens directly by overriding certain tokens.
+### Extending an existing `ThemeReference`
 
-### Components
+We have a concept of `ThemeRecipes` which allow for layering of partial theme objects to create the ultimately desired theme. `ThemeRecipes` are functions which take a `Theme` and spit out a `PartialTheme`, which is then deep merged into the base them object.
```

Comment:
Is there a definition of a PartialTheme anywhere that we can link? I think it's pretty self-explanatory, but I'm not 100% sure I know everything about it

Reply:
Yeah I can link to the theme.types page here, thanks!